src/Container.ts

```diff
@@ -131,6 +131,16 @@ export class Container {
         });
     }
 
+    /**
+     * Clears the container by removing all previously registered artifacts from it.
+     */
+    static clear () {
```

Comment:
I've renamed it to `reset`. I think individual method should be called `remove()` and it's outside of the scope of this PR. Probably, we should create another issue for it.